source/common/http/http1/codec_impl.h

```diff
@@ -35,10 +35,19 @@ namespace Http1 {
   COUNTER(requests_rejected_with_underscores_in_headers)                                           \
   COUNTER(response_flood)
 
+// Constructor-args for CodecStats.
+#define HTTP1_CODEC_STATS(scope) ALL_HTTP1_CODEC_STATS(POOL_COUNTER_PREFIX(scope, "http1."))
```

Comment:
Is the macro needed? Can you just inline this below? Same for h2?

Reply:
It was used in tests, but making stats_ a reference in codec_impl.cc centralized the storage for these to always be via AtomicPtr, so now there's just one place these are constructed.